runtime/rococo/src/lib.rs

```diff
@@ -171,7 +171,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("bitgreen-rococo"),
```

Comment:
should not be replace "rococo" string?

Reply:
we have two runtimes, one for rococo and one for polkadot https://github.com/bitgreen/bitgreen-node/pull/185/files#diff-3aa95c6867fdabb806fb1f2890a2affc820ce25ae00e066614fd01d00b82a725